lib/helper/Puppeteer.js

```diff
@@ -148,8 +147,8 @@ const consoleLogStore = new Console();
 class Puppeteer extends Helper {
   constructor(config) {
     super(config);
-
-    puppeteer = requireg(config.browser === 'firefox' ? 'puppeteer-firefox' : 'puppeteer');
+    
+    puppeteer = requireg(config.browser === 'firefox' ? 'puppeteer-firefox' : (config.chrome.browserWSEndpoint ? 'puppeteer-core' : 'puppeteer'));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not nest ternary expressions. (no-nested-ternary)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4646938)